### PR TITLE
Distpg-71 release notes 12.2

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by borys-belinsky — APPROVED** [review]

File: source/installing.rst [hunk 1]
```diff
@@ -19,11 +19,11 @@ Like many other |percona| products, |pdp| is installed by using the
    <https://www.percona.com/doc/percona-repo-config/percona-release.html#percona-release-update-latest-version>`_
    of |percona-release| for details.
 
-As soon as |percona-release| is up-to-date, *set up* the |pdp| product (``ppg-11``).
+As soon as |percona-release| is up-to-date, *set up* the |pdp| product (``ppg-12``).
 
 .. code-block:: bash
 
-   $ sudo percona-release setup ppg11
+   $ sudo percona-release setup ppg12
```
Comment (EvgeniyPatlan) [inline source/installing.rst]: just a question.
We are going to issue ppg-12.2 but why we do not show how to install this distribution but generic only?

Comment (EvgeniyPatlan) [reply source/installing.rst]: This comes from that we have special repo for each distribution and that is why this format doesn't suit for distributions

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]
we need to show that ppg-12.2 should be used.
ppg-12 is generic repo and it just accumulates all pgg-12 packages
the same about ppg-11

File: source/installing.rst [hunk 2]
```diff
@@ -19,39 +19,40 @@ Like many other |percona| products, |pdp| is installed by using the
...
    of |percona-release| for details.
 
-As soon as |percona-release| is up-to-date, *set up* the |pdp| product (``ppg-11``).
+As soon as |percona-release| is up-to-date, *set up* the required version of the
+|pdp| product (``ppg-<version>``). For example, to set up |pdp| 12.2, use the following command:
 
 .. code-block:: bash
 
-   $ sudo percona-release setup ppg11
+   $ sudo percona-release setup ppg12.2
```
Comment (EvgeniyPatlan) [inline source/installing.rst]: there are 2 ways to call setup with and without dash
percona-release setup ppg12.2 or percona-release setup ppg-12.2
as for me it is better to use variant with dash

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**